### ISSUE TITLE
Tags overflow horizontally off page in IE11

### ISSUE THREAD

**Issue by chrisbmoore** [Gbuomprisco/ng2-tag-input#390]
**I'm submitting a ...**  (check one with "x")
```
[X] bug report => search github for a similar issue or PR before submitting
[ ] feature request
[ ] support request/question
```

**Current behavior**

Tags start to overflow horizontally beyond the desired parent element or page when over 50 or so of varying length in IE11.

**Expected behavior**

Tags should always remain inside parent element, and only push down to fit. This works correctly in Chrome.

**Minimal reproduction of the problem with instructions (if applicable)**

In IE11, go to http://www.buompris.co/ng2-tag-input/ and add 50 or more tags with varying lengths.

**What is the motivation / use case for changing the behavior? (if applicable)**

Doesn't look good and can't read tags easily off the edge of the page.

**What do you use to build your app? (SystemJS, Webpack, angular-cli, etc.). Please specify the version**

SystemJS 0.19.41

**Angular version:** 

Angular 4.1.0

**ng2-tag-input version:**

1.2.0. Using this version as other errors happen with newer versions during build/runtime. Possibly due to any recent module changes. That is for another issue.

**Browser:** [all | Chrome XX | Firefox XX | IE XX | Safari XX | Mobile Chrome XX | Android X.X Web Browser | iOS XX Safari | iOS XX UIWebView | iOS XX WKWebView ]

IE 11.0.96

**Comment by chrisbmoore** [comment]
I've looked at the styling, and the formatting appears to be fixed when I change **.ng2-tag-input.bootstrap** to have **display:inline** instead of **display:flex**. The results seem consistent with IE 11, Chrome and Firefox.

I've never made a Pull Request before and don't have the ng2-tag-input project setup for building, so if you wanna go ahead and try that CSS change, that'd be helpful @Gbuomprisco.

**Comment by realappie** [comment]
@chrisbmoore Why not create your first PR! It will be easy, I promise.

**Comment by Gbuomprisco** [comment]
@chrisbmoore I can try that. Or if you wanna make a PR (like @realappie said), we can help you out make it :) contributors are very welcome

**Comment by chrisbmoore** [comment]
@realapppie @Gbuomprisco Sure, it's a good opportunity for me to make a PR. I'll try my hand at it later today or tomorrow.

**Comment by Gbuomprisco** [comment]
Published v1.2.8
